src/Engine/ProtoCore/RuntimeStatus.cs

```diff
@@ -70,35 +72,50 @@ public IOutputStream WebMessageHandler
             set;
         }
 
-        public IEnumerable<Runtime.WarningEntry> Warnings
+        public List<Runtime.WarningEntry> Warnings
```

Comment:
this should be ok.
SHould **not** be a breaking change.

Reply:
How is this not a breaking change?

Reply:
last sentence says the signature should not change.

Reply:
@mjkkirschner I thought the return type is not included in the method signature

Reply:
```
A return type of a method is not part of the signature of the method for the purposes of method overloading. However, it is part of the signature of the method when determining the compatibility between a delegate and the method that it points to.
```

Reply:
So if somebody created a delegate that points to the getter/setter of the `Warnings` ..that might break
not sure if we should worry about this scenario

Reply:
The only thing that quote indicates a signature does not include the return type, is when overloading - we're not overloading here, and it doesn't mention all the places that the signature does include the return type.

Reply:
Is there any issue here with just keeping it IEnumerable?